_test/mono_pkg.yaml

```diff
@@ -1,5 +1,5 @@
 dart:
-- dev
```

Comment:
Oh i think we actually want to leave these at `-dev` - we don't generally test the `beta` branch. You could argue we should but that is a separate discussion :).

Reply:
Doing so breaks all tests until the analyzer update is complete, also.  Followup will revert this.

Reply:
Huh i am confused by how it would break the tests? But if we just change back later that should be fine.

Reply:
analyzer 0.42 is not compatible with 2.13 dev

Reply:
Oh, its that it picks up 2.13 dev (and not 2.12 dev). Got it.